plugin-packs/postcss-preset-env/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Unreleased (unknown)
```

Comment:
Should this be changed to minor? Not sure how the release will work through this

Reply:
Yes, I've changed this